src/chocolatey.resources/helpers/functions/Install-ChocolateyShortcut.ps1

```diff
@@ -108,12 +127,39 @@ directoy, an icon to be used for the shortcut, along with a description and argu
 		if ($description) {
 		  $lnk.Description = $description
 		}
-	   
+    if ($WindowStyle) {
+      $lnk.WindowStyle = $WindowStyle
+    }
+
 	    $lnk.Save()
-		
+
 		Write-Debug "Shortcut created."
+
+    [System.IO.FileInfo]$Path = $shortcutFilePath
+
+    If ($RunAsAdmin) {
+      $TempFileName = [IO.Path]::GetRandomFileName()
+      $TempFile = [IO.FileInfo][IO.Path]::Combine($Path.Directory, $TempFileName)
+      $Writer = New-Object System.IO.FileStream $TempFile, ([System.IO.FileMode]::Create)
+      $Reader = $Path.OpenRead()
+      While ($Reader.Position -lt $Reader.Length) {
+        $Byte = $Reader.ReadByte()
+        If ($Reader.Position -eq 22) {$Byte = 34}
+        $Writer.WriteByte($Byte)

```

Comment:
This bit of code is not mine - but a more or less "community developed" method of editing the shortcut's binary data to flip a bit because there is no API to actually set the "Run As Admin" checkbox.

Here are some links - but since it is a method ported from before PowerShell, it's hard to track down origination.

Let me know if you want some of these links in an updated commit.

http://stackoverflow.com/questions/28997799/how-to-create-a-run-as-administrator-shortcut-using-powershell

http://stackoverflow.com/questions/9701840/how-to-create-a-shortcut-using-powershell

http://poshcode.org/2513